workflow/scripts/copy_files.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python3
+
+import pandas as pd
+import os
+import yaml
+
+try:
+    output = snakemake.params.output
+except NameError:
+    with open('config/config.yaml', 'r') as f:
+        config = yaml.safe_load(f)
+    output = config['output']
```

Comment:
When is `snakemake.params.output` supposed to be filled? Will this not succeed with an empty string for the param?